File: Palindrom/palindrome.py
# Brute force method
# Time and space complexity is O(n)
s = "racecar"

def isPalindrome(s):
    rev = ""
    for i in range (len(s)-1,-1,-1,-1):
        rev += s[i]
    if(rev == s):
        return True
    else:
        return False

result = isPalindrome(s)

File: Palindrom/palindrome.js
// Brute force method

function isPalindrome(arr){
    let rev = []
    for(let i=arr.length-1; i>0; i--){
        rev.push(arr[i])
    }
    
    if(rev === arr){
        return true
    }
    else{
        return false
    }
}